envs/strobealign.yml

```diff
@@ -0,0 +1,6 @@
+name: strobealign
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - strobealign
```

Comment:
Dont you want to tie this to a specific version so that the behavior is reproducible?